acceptance-tests/src/test/resources/uk/gov/di/test/features/Strategic_App.feature

```diff
@@ -0,0 +1,53 @@
+
+  # ENTER INCORRECT PASSWORD TOO MANY TIMES DURING SIGN IN - 2060
+  Scenario: A user is blocked when they enter too many incorrect passwords during sign in.
+    Given a user exists
+    When the user comes from the stub relying party with option Strategic-App and is taken to the "Sign in to GOV.UK One Login" page
+    And the user selects sign in
+    Then the user is taken to the "Enter your email" page
+    When the user enters their email address
+    Then the user is taken to the "Enter your password" page
+    When the user enters an incorrect password 6 times
+    Then the "You entered the wrong password too many times" lockout screen is displayed
+    And the "wait 2 hours, then try again" lockout Text is displayed
+
+    * the lockout duration is 2 hours
+    * the lockout reason is "you entered the wrong password"
+
+    Given the lockout has not yet expired
+    When the user comes from the stub relying party with option Strategic-App and is taken to the "Sign in to GOV.UK One Login" page
+    And the user selects sign in
+    Then the user is taken to the "Enter your email" page
+    When the user enters their email address
+    Then the "You cannot sign in at the moment" lockout screen is displayed
+    And the "[Reset your password, ., , You can also wait 2 hours, then try again.]" retry Text is displayed
+    * the lockout duration is 2 hours
+    * the lockout reason is "you entered the wrong password"
+
+  Scenario: A user is blocked when they enter too many incorrect sms codes during sign in.
+    Given a user with SMS MFA exists
+    When the user comes from the stub relying party with option Strategic-App and is taken to the "Sign in to GOV.UK One Login" page
+    And the user selects sign in
+    Then the user is taken to the "Enter your email" page
+    When the user enters their email address
+    Then the user is taken to the "Enter your password" page
+    When the user enters their password
+    Then the user is taken to the "Check your phone" page
+    When the user enters an incorrect phone security code 6 times
+    Then the "You entered the wrong security code too many times" lockout screen is displayed
+    And the "Wait 2 hours, then try again." lockout Text for SMS is displayed
+    * the lockout duration is 2 hours
+
+    Given the lockout has not yet expired
+    When the user comes from the stub relying party with option Strategic-App and is taken to the "Sign in to GOV.UK One Login" page
+    And the user selects sign in
+    Then the user is taken to the "Enter your email" page
+    When the user enters their email address
+    Then the user is taken to the "Enter your password" page
+    When the user enters their password
+    Then the "You cannot sign in at the moment" lockout screen is displayed
+    And the "Wait 2 hours, then try again." lockout Text for SMS is displayed
+    * the lockout duration is 2 hours
+    * the lockout reason is "you entered the wrong security code too many times"
```

Comment:
Hi @Vivektiwari11 - I might add a newline at the end of this file.

Reply:
Hi @gtvj I have just removed last line.  Please check

Reply:
Hi @Vivektiwari11 - sorry, I meant to _add_ a newline (so as to avoid the "No newline at end of file" warning).

Reply:
Hi @Vivektiwari11 - just one more thing to mention. Do you think there might be benefit it reducing the number of commits. Looking at the changes, it feels like this could be introduced via one or perhaps two commits

Reply:
Hi @gtvj will make sure in future to keep the commit at minimum.